zappa/wsgi.py

```diff
@@ -100,7 +93,8 @@ def create_wsgi_request(
     if "," in x_forwarded_for:
         # The last one is the cloudfront proxy ip. The second to last is the real client ip.
         # Everything else is user supplied and untrustworthy.
-        remote_addr = x_forwarded_for.split(", ")[-2]
+        addresses = [addr.strip() for addr in x_forwarded_for.split(",")]
+        remote_addr = addresses[-2]  # get client ip
```

Comment:
```suggestion
        remote_addr = addresses[-2]
```
remove duplicate comment, is explained in Line 94

Reply:
removed!